airbyte-server/src/main/java/io/airbyte/server/converters/SchemaConverter.java

```diff
@@ -24,107 +24,87 @@
 
 package io.airbyte.server.converters;
 
-import io.airbyte.api.model.SourceSchema;
-import io.airbyte.api.model.SourceSchemaField;
-import io.airbyte.api.model.SourceSchemaStream;
+import io.airbyte.api.model.AirbyteStreamConfiguration;
+import io.airbyte.api.model.SyncMode;
 import io.airbyte.commons.enums.Enums;
 import io.airbyte.commons.text.Names;
-import io.airbyte.config.DataType;
-import io.airbyte.config.Field;
-import io.airbyte.config.Schema;
-import io.airbyte.config.Stream;
 import java.util.List;
 import java.util.stream.Collectors;
 
-// todo (cgardens) - update this before merging.
+/**
+ * Convert classes between io.airbyte.protocol.models and io.airbyte.api.model
+ */
 public class SchemaConverter {
 
-  public static Schema toPersistenceSchema(SourceSchema sourceSchema) {
-    final List<Stream> persistenceStreams =
-        sourceSchema.getStreams()
-            .stream()
-            .map(apiStream -> {
-              final List<Field> persistenceFields = apiStream.getFields()
-                  .stream()
-                  .map(SchemaConverter::toPersistenceField)
-                  .collect(Collectors.toList());
+  public static io.airbyte.api.model.AirbyteStream convertTo(final io.airbyte.protocol.models.AirbyteStream stream) {
```

Comment:
i think these names need to include more context. we are converting back and forth between 2 models with the same name and then the other one converts in the other direction. this is hard to parse as a human if they are just called `convertTo`. it is super easy to mess up.

i would suggest `toApi` and to `toBackend`. Not married to these exact names, but we need something to tell us what "direction" the conversion is happening in.

Reply:
the names you use in your tests are also fine.

Reply:
Ok